ocs_ci/ocs/managedservice.py

```diff
@@ -278,3 +278,30 @@ def get_admin_key_from_provider():
     finally:
         config.switch_ctx(initial_cluster_index)
         return admin_key
+
+
+def check_default_cluster_context_index_equal_to_current_index(switch_to_default=True):
```

Comment:
The name of the function contains `check*` string. I suggest not to set `switch_to _default` to `True` as it could be confusing because the function name suggests only check and not switching or setting any config value.

Reply:
The thing is that I want to change the current index to the default index if they are not equal.

Reply:
I split it into three small functions.